hosts

```diff
@@ -95,10 +95,10 @@ builder7.rht.gluster.org
 builder8.rht.gluster.org
 builder9.rht.gluster.org
 
-
 [jenkins_builders:children]
 jenkins_builders_rht
 jenkins_builders_rackspace
+bugziller
```

Comment:
This mean this is gonna setup a whole jenkins builder env, ie, everything to test glusterfs, is that also what we want ?